composer.json

```diff
@@ -28,6 +28,7 @@
 	"require-dev": {
 		"nette/bootstrap": "~2.3",
 		"nette/tester": "~2.0.0",
+		"marc-mabe/php-enum": "~3.0",
```

Comment:
Love this choice

Reply:
This is only for testing purposes, but iml is now pretty simple :)